### PR TITLE
Unread get channels

### PR DESCRIPTION
**Pull request by basafilm**
# Description

To count and update unread messages needed to fetch channels . when a channel fetched it count the number of unread messages and when open the channel it says " the channel is open" that means the unread messages become read. The channels fetch based on channel id and can match with unread table which also fetch based on channelId and userId. 

Fixes # (issue)
some related links placed on UnreadMessages component . storybook error is because of those links. the two;UnreadMessages and overviewChannel should test together. 
.../overview 
.../overview/id 

# Checklist

- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/GetChannels/GetChannels.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, Link } from 'react-router-dom';
+
+function GetChannels() {
```
Comment (tpetrina) [inline src/client/components/GetChannels/GetChannels.js:4]: If you could rename this component to `OverviewChannels`, that would be slightly better name.

**Review by tpetrina — COMMENTED** [review]
Couple of small fixes here and there.

**Review by tpetrina — COMMENTED** [review]
You have extra code and some typos.

**Comment by basafilm** [comment]
No need anymore to this

**Pull request closed without merging**